frontends/web/src/locales/en/app.json

```diff
@@ -313,7 +313,43 @@
       "noExchanges": "Sorry, there are no available exchanges in this region.",
       "region": "Region",
       "selectRegion": "Select region",
-      "title": "Buy {{name}}"
+      "title": "Buy {{name}}",
+      "infoContent": {
+        "region": {
+          "title": "Select the region your bank account is registered in to see which options are available to you."
+        },
+        "moonpay": {
+          "supportedCurrencies": "Supports all major fiat currencies: USD, EUR, CHF, and more.",
+          "fullCurrenciesList": "See full list of currencies here",
+          "payment": {
+            "title": "Payment methods",
+            "creditDebitCard": "Credit/debit Card",
+            "creditDebitCardDetails": {
+              "cards": "Amex, Mastercard, Visa and Maestro",
+              "applePayAndroidPay": "Apple Pay/Android Pay"
+            },
+            "bankTransfer": "Bank transfer*",
+            "bankTransferDetails": {
+              "sepa": "SEPA and SEPA Instant (EUR transactions in SEPA countries only)",
+              "uk": "UK Faster Payments (GBP transactions in the UK only)",
+              "pix": "PIX (BR transactions in Brazil only)"
+            },
+            "asteriskText": "* Not available for US residents",
+            "learnMore": "See more details about payment methods"
+          },
+          "fees": {
+            "title": "Fees",
+            "creditDebitCard": "Credit/debit card: 4.9%",
```

Comment:
It could be nice to avoid the numbers and load them dynamically from the deals instead. wdyt? No strong opinion though

Reply:
Ah, sure would be possible. Where are we getting these values from?

Reply:
They are collected by `getExchangeDeals` https://github.com/digitalbitbox/bitbox-wallet-app/blob/staging-pocket/frontends/web/src/routes/buy/exchange.tsx#L68

Reply:
yup, just realised :)

Reply:
> It could be nice to avoid the numbers and load them dynamically from the deals instead. wdyt? No strong opinion though

and yes, that'd be great!